packages/shared/lib/chart.ts

```diff
@@ -42,17 +43,30 @@ export enum AccountChartType {
 
 const BAR_CHART_ACTIVITY_MONTHS = 6
 
-/** Selected currency on chart */
-export const chartCurrency = writable<CurrencyTypes>(CurrencyTypes.USD)
+export interface ChartSelectors {
+    currency: AvailableExchangeRates,
+    timeframe: HistoryDataProps
+}
+export interface Chart {
+    time: number
+    type: string,
+    message: string
+}
 
-/** Selected time frame on chart */
-export const chartTimeframe = writable<HistoryDataProps>(HistoryDataProps.SEVEN_DAYS)
+/** Selected currency and timeframe on chart */
+export const chartSelectors = persistent<ChartSelectors>('chartSelectors', { currency: AvailableExchangeRates.USD, timeframe: HistoryDataProps.SEVEN_DAYS })
```

Comment:
These should probably persist in the profile not directly in the storage, otherwise different logins will pickup the same settings.

Reply:
> Actually I guess this is fixed by #770
Correct, that PR should fix it

Reply:
will persist it in settings 👍🏼

Reply:
@obany updated